Client/WebFilters/AdblockResourcesMappings.swift

```diff
@@ -5,7 +5,14 @@
 import Foundation
 
 struct AdblockResourcesMappings {
-    static let generalAdblockName = "latest"
+    static func generalAdblockName(for fileType: FileType) -> String? {
+        switch fileType {
+        case .dat: return "rs-ABPFilterParserData"
```

Comment:
Some logic had to be added to handle differences between ios-json and universal-dat files, this includes different endpoints and file names